spec/test-data/translations/glotpress-download/expected/values/strings.xml

```diff
@@ -45,6 +45,29 @@
         <item>All Orders</item>
     </string-array>
 
-    
+
     <string name="shipping_label_payments_cant_edit_warning">Only the site owner can manage the shipping label payment methods. Please contact Store Owner %1$s (%2$s) to manage payment methods.</string>
+    <string name="threat_fix_description">Pricing ranges from $70–120/hour</string>
+    <string-array name="settings_jetpackdescription">
+        <item>You may mark an IP address (or series of addresses) as \"Always allowed\"</item>
+        <item>preventing them from ever being blocked by Jetpack. IPv4 and IPv6 are acceptable.</item>
+        <item>To specify a range, enter the low value and high value separated by a dash. Example: 12.12.12.1–12.12.12.100</item>
+    </string-array>
+    <string name="threat_fix_description_large">Pricing ranges from $70 – 120/hour</string>
```

Comment:
What about if the string is `$70 - $120` instead of `$70 - 120`? worth adding a test case for that imho